tests/unit/async_db/test_connection.py

```diff
@@ -191,9 +187,7 @@ async def test_connect_engine_name(
 
     # Mock engine id lookup by name
     httpx_mock.add_response(
-        url=f"https://{settings.server}"
-        + ACCOUNT_ENGINE_ID_BY_NAME_URL.format(account_id=account_id)
-        + f"?engine_name={engine_name}",
+        url=get_engine_id_by_name_url + f"?engine_name={engine_name}",
```

Comment:
Only thing, I'd move `"?engine_name={engine_name}"` to the `get_engine_id_by_name_url` fixture, as originally suggested.